test/service.thrift

```diff
@@ -0,0 +1,7 @@
+
+service Foo {
+    byte foo(0: byte number) throws (
```

Comment:
Is this considered valid? Thrift doesn't allow field identifier 0 to be used my service authors. It's reserved for the return type in the response union.